documentation/library_guide/pstl/dpcpp_policies_usage.rst

```diff
@@ -18,11 +18,12 @@ Follow these steps to add Parallel STL to your application:
    namespace, to a parallel algorithm.
 #. Use the C++ Standard Execution Policies:
 
-   * Compile the code with options that enable OpenMP* vectorization pragmas.
-   * Link with the |onetbb_long| or |tbb_long| dynamic library for parallelism.
+   #. Compile the code with options that enable OpenMP* vectorization pragmas.
```

Comment:
Does Sphinx recognize this is a numbered list embedded in a list and use additional numbering?  For example, assuming the `#.` on line 20 is the first item of the outer list, would the items in this list be numbered `1.1, 1.2, 1.3`?  If it doesn't then using a different item marker style might be easier to read.

Reply:
Currently, it defaults to 1, 2, 3 and does the same for the nested steps. I am looking for a longer term solution that reads easier, but that will be for the next release. I thought this was better than a mixed number/bullet list, which it was before.

Reply:
Checking the document currently published it looks like there was some manual modifications because the bulleted list is bullets in one sublist and lowercase letters in another.  I agree that numbers or letters look better than the bullets.  I think the indent is large enough and the sublists are short so the restart in numbering won't be too confusing in this release. Ok to merge.